test/aaa_profiling/test_memusage.py

```diff
@@ -1051,6 +1058,12 @@ def test_many_discarded_relationships(self):
         """a use case that really isn't supported, nonetheless we can
         guard against memleaks here so why not"""
 
+        import sysconfig
```

Comment:
not needed, use `gil_enabled` req on test_many_discarded_relationships

e.g. for this one and the rest:

```
@testing.requires.gil_enabled
def test_many_discarded_relationships()
```